src/syscall/js/js.go

```diff
@@ -580,4 +604,5 @@ func CopyBytesToJS(dst Value, src []byte) int {
 	return n
 }
 
+//go:noescape
```

Comment:
Justification: this will affect whether byte slices with dynamic contents but statically known length to not escape anymore.